css/home.css

```diff
@@ -133,7 +133,7 @@ li a:hover
 
 @media(max-width: 700px){
   *{
-    max-width: max-content;
+    max-width: 100vw;
```

Comment:
this (*) will make every element be 100% of the view width. this is why your nav bar links in the home page won't line up. they're each 100% vw long.